server/services/telegram/lib/message.new.js

```diff
@@ -30,8 +31,10 @@ async function newMessage(msg) {
     user_id: user.id,
     user,
     language: user.language,
-    date: msg.date,
+    created_at: new Date(msg.date * 1000).toISOString(),
     text: msg.text,
+    // We generate ID as uuidv4 as Telegram doesn't give this type of ID
+    id: uuid.v4(),
```

Comment:
Do we really an UUID v4 ID, telegram provides a message_id, which is an integer: https://core.telegram.org/bots/api#message

Reply:
We cannot mix UUIDv4 and int in Database
https://github.com/GladysAssistant/Gladys/blob/master/server/models/message.js#L6

Reply:
So we store telegram messages in database? We don't delegate it to telegram?

Reply:
Exactly. We saved a copy (of user and gladys messages).

Reply:
yes we save them in the database